### ISSUE TITLE
position of caret in <add rend="caret"> on base line

### ISSUE THREAD

**Issue by wehooper** [IUBLibTech/newton_chymistry#47]
This is a tweak of the caret, which is appearing now where it should.

We would like to have the caret on the baseline in the same font as the main text and then have the \<add\> content raised and decreased in size.

There's an example at the top of "Nature's Obvious Laws" ALCH00081xml, Dibner 1031 B (today's document) in item number two (the one not struck out), search 'Proved'.
P4: http://carbon.dlib.indiana.edu:8215/newton-dev/mss/dipl/ALCH00081/
P5: http://carbon.dlib.indiana.edu:8220/text/ALCH00081/diplomatic#?c=&m=&s=&cv=&xywh=-1%2C-1%2C2%2C1

In ALCH00081.xml, see line 402.

**Comment by Conal-Tuohy** [comment]
How's that? 
http://carbon.dlib.indiana.edu:8220/text/ALCH00081/diplomatic?highlight=transitorily

**Comment by wehooper** [comment]
Great, it's a wrap.

**Comment by tubesoft** [comment]
It seems to work when I put the caret rendition part inside `<xsl:when test="$view = 'diplomatic' ">` like this:

`p5-to-html.xsl`
BEFORE

https://github.com/IUBLibTech/newton_chymistry/blob/34e11fbada5307c57598b1b6180703206e39ac9b/xslt/p5-to-html.xsl#L296-L309

AFTER
```
		<xsl:choose>
			<xsl:when test="$view = 'diplomatic' ">
				<xsl:if test="tokenize(@rend) = 'caret'">
				      <xsl:text>‸</xsl:text>
				</xsl:if>
				<xsl:element name="ins">
					<xsl:apply-templates mode="create-attributes" select="."/>
					<xsl:apply-templates mode="create-content" select="."/>
				</xsl:element>
			</xsl:when>
			<xsl:otherwise>
				<xsl:apply-templates mode="create-content" select="."/>
			</xsl:otherwise>
		</xsl:choose>
```

**Comment by tubesoft** [comment]
Fixed at #104

**Comment by wehooper** [comment]
This fix is working, the carets are appearing again.
